src/intl/en/page-developers-docs.json

```
@@ -73,8 +73,8 @@
   "docs-nav-scaling-sidechains": "Sidechains",
   "docs-nav-scaling-plasma": "Plasma",
   "docs-nav-scaling-validium": "Validium",
-  "docs-nav-security": "Security",
-  "docs-nav-security-description": "Best practices for managing smart contract attacks and vulnerabilitie",
+  "docs-nav-smart-contract-security": "Smart Contract Security",
```

```suggestion
  "docs-nav-smart-contract-security": "Smart contract security",
```
Changing to sentence case for consistency w/ the rest of our links here.